server.js

```diff
@@ -1,22 +1,21 @@
 (function(global) {
 
 const HERE_FILE = 'here.txt';
```

Comment:
why even have this if it never changes?  and if you remove this, remove `read()` too.